platform/darwin/src/MGLShapeSource.h

```diff
@@ -305,6 +307,10 @@ MGL_EXPORT
  */
 - (NSArray<id <MGLFeature>> *)featuresMatchingPredicate:(nullable NSPredicate *)predicate;
 
+// TODO: doc
+- (NSArray<id <MGLFeature>> *)leavesOfCluster:(id<MGLPointCluster>)cluster offset:(uint32_t)offset limit:(uint32_t)limit;
```

Comment:
As with above, curious about the necessity of `uint32_t` here.

Reply:
These I think we could change without too much concern.